src/layman/common/util.py

```diff
@@ -12,3 +12,12 @@ def merge_infos(infos):
         for (name, info) in source.items():
             result_infos[name].update(info)
     return result_infos
+
+
+def clear_publication_info(info):
```

Comment:
Test that GET Layer|GET Map responses do not contain id and type keys. E.g. in src/layman/layer/rest_test.py, line 312
```
layer_info = client.get(url_for('rest_layer.get', username=username, layername=layername)).get_json()
```